core/templates/dev/head/pages/get_started/get_started.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}I18N_GET_STARTED_PAGE_TITLE{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+
+  <style>
+    html, body {
+      background-color: #eee;
+    }
+  </style>
+{% endblock %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      About
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div class="oppia-about-header">
+    <h1>Get Started</h1>
+  </div>
+
+    <div class="oppia-page-card oppia-about-content">
+    
+    
+    <div class="oppia-about-card-content-wide">
+      
+      <p>
+        Creating an Oppia lesson is easy and free.  After you create the lesson, you share it with 
+        others, and get feedback you can use to improve the lesson.  </p>
+        
+      <h3>Choose a Topic</h3>
+      <p>All you need to get started is a topic you want 
+        to teach.  You can create a lesson about any topic, large or small.  The ideal size of 
+        topic for a lesson is one that you would cover in a single 
+        class.  You can also create multiple related lessons that are intended to be completed 
+        in sequence.
+      </p>
+        
+      
+      <h3>Create Your Lesson</h3>
+      <p>When you have chosen a topic, just click ‘Create’, and log in with your Google account.    
+      If you don't have a Google account, you can <a href="https://accounts.google.com/SignUp" target="new">create one here</a>.</p>
+      <p>A lesson consists of a number of steps.  A step can can include text (e.g. a written 
+      explanation), pictures and video.  Each step presents the student with a question, which 
+      they must attempt in order to continue. It can be a multiple choice question, or require 
+      them to type something in, or any one of a number of other available interactions.</p>
+      <p>Once the learner has answered the question, Oppia will give them feedback and let them 
+      continue to the next step.  To see how learners experience Oppia, try out one of these lessons:</p>
+      <p>
+      <ul>
+      <li><a href="https://www.oppia.org/explore/0" target="new">Welcome to Oppia</a></li>
+      <li><a href="https://www.oppia.org/explore/N9iXI-fZTo8s" target="new">Introduction 
+      to Chili Peppers</a></li>
+      <li><a href="https://www.oppia.org/explore/9" target="new">Pitch Perfect</a></li>
+      </ul>
```

Comment:
Indent by 2. (Note that the point of all this is to make the tree structure clearer.)